httpserver-rs/README.md

```diff
@@ -1,20 +1,20 @@
-# wasmcloud HTTP Server Provider
+# wasmCloud HTTP Server Provider
 
-This library is a _native capability provider_ for the `wasmcloud:httpserver` 
-capability. Only actors signed with this capability 
-privilege will be allowed to use it. The actor that implements this contract 
-must implement the [HandleRequest](https://docs.rs/wasmcloud-interface-httpserver/0.1.5/wasmcloud_interface_httpserver/trait.HttpServer.html#) operation
+This executable is an implementation of the `wasmcloud:httpserver` capability. Only actors signed with this capability contract ID will be allowed to use it. 
```

Comment:
This capability provider implements the `wasmcloud:httpserver` capability contract, and enables an actor to accept incoming http(s) requests. It is implemented in Rust with the [warp](https://docs.rs/warp/) web server framework and the fast and scalable [hyper](https://docs.rs/hyper/) http implementation.